pages/[network]/avatar/[address].tsx

```diff
@@ -119,21 +124,22 @@ export const getServerSideProps: GetServerSideProps<AvatarPageProps> = async (
 
     const nodeType = getNodeType(network);
     const networkType = getNetworkType(network);
-    const avatarJsonObj = await getAvatar(nodeType, networkType, address);
-
     const sdk = getSdk(networkType, nodeType);
-    const inventoryJsonObj = await sdk.inventory(address);
+    const avatarJsonObj = await sdk.avatar(address);
+    const inventoryJsonObj = avatarJsonObj.inventory;
     const inventoryObj = parseToInventory(inventoryJsonObj);
-    const balanceJsonObjs = await Promise.all(
-        CURRENCY_TICKERS.map(
-            (currencyTicker, index) => getBalance(
-                nodeType,
-                networkType,
-                index <= 0 ? avatarJsonObj.agentAddress : address,
-                currencyTicker
+    const balanceJsonObjs = avatarJsonObj.agentAddress === null
+        ? []
+        : await Promise.all(
+            CURRENCY_TICKERS.map(
+                (currencyTicker, index) => getBalance(
+                    nodeType,
+                    networkType,
+                    index <= 0 ? avatarJsonObj.agentAddress : address,
```

Comment:
I don't understand `index <= 0` condition. Previously, it was a code like the below:

```typescript
const fetchedFavs = await Promise.all(
    CURRENCIES.map(
        (currency) => sdk.GetBalance({
            currency: currency,
            address: currency.ticker === "CRYSTAL" ? avatarJsonObj.agentAddress : address,
            hash: hash,
        })
    )
);
const favs = fetchedFavs.map((resp, index) => {
    return {
        ticker: CURRENCIES[index].ticker,
        amount: parseFloat(resp.stateQuery.balance.quantity),
    };
});
```

It may need to be:
```suggestion
                    currencyTicker === "CRYSTAL" ? avatarJsonObj.agentAddress : address,
```

Reply:
When I modified this, only "CRYSTAL" is currently the only currency bounded to agents, but I've considered adding more in the future.
For example, the first few `index`s in the `CURRENCY_TICKERS` array are for the agent.
But this is an undecided specification, there are other ways to implement it, and you made a good suggestion, so I'll accept it.🙂

Reply:
@moreal Review 9c0ace1 please~!